elastalert/elastalert.py

```diff
@@ -456,8 +456,21 @@ def get_hits_count(self, rule, starttime, endtime, index):
             five=rule['five']
         )
 
+        es_client = self.thread_data.current_es
         try:
-            res = self.thread_data.current_es.count(index=index, doc_type=rule['doc_type'], body=query, ignore_unavailable=True)
```

Comment:
Was sending `doc_type` regardless.  In 7.x `doc_type` was effectively removed.  There's an implicit `_doc` type, but this doesn't align with our 6.x cluster.